dll/win32/msi/dialog.c

```diff
@@ -3159,8 +3159,22 @@ static void msi_dialog_vcl_add_columns( msi_dialog *dialog, msi_control *control
          * if a width is invalid, all remaining columns are hidden
          */
         if ( !wcsncmp( num, L"-", 1 ) || !str_is_number( num ) ) {
+#ifdef __REACTOS__
+            if ((count == 0) && (!wcsncmp( num, L"\\", 1 ) || !wcsncmp( num, L"&", 1 )))
```

Comment:
@KRosUser2 Looking at your "better fix" patch in the WineHQ Bugzilla, that one has the redundant bFirst variable, and also the bug that you call `msi_free(num);` before having those `wcsncmp(num, ...)` calls. As such I think it should be instead fixed with something as being done in this PR.